src/constants/UrlAndParameters.js

```diff
@@ -3,6 +3,7 @@ const URL_AND_PARAMETERS = {
     COLLECTION: "york-uni-courses",
     FORM: "course-search",
     PROFILE: "_default_preview",
+    SMETA_CONTENT_TYPE: "course",
```

Comment:
```suggestion
    CONTENT_TYPE: "course",
```
I think the name `smeta_contentType` is weird enough that it should also be a constant here, although at that point would it be better to have an object with default keys and values that the UrlConstructor loops through?

Reply:
re: 'would it be better to have an object with default keys and values that the UrlConstructor loops through?' Does sound a better idea

Reply:
I agree, but it doesn't have to be done in this ticket so I've accepted these changes

Reply:
However, I have attempted it, and it actually makes the tests especially much harder to understand - you are basically using virtually the same logic to construct the url you are expecting to get as is used in constructFunnelbackUrls.js itself, and it's not very readable.

Reply:
I don't want to rename smeta_contentType, as it reflects the actual parameter name, like all the others do. That name has been dictated by Funnelback.